src/modelmanager.cpp

```diff
@@ -1468,6 +1485,11 @@ Status ModelManager::createPipeline(std::shared_ptr<MediapipeGraphExecutor>& gra
     const std::string& name,
     const KFSRequest* request,
     KFSResponse* response) {
+#if (MEDIAPIPE_DISABLE == 0)
     return this->mediapipeFactory.create(graph, name, request, response, *this);
+#else
+    SPDLOG_ERROR("Mediapipe support was disabled during build process...");
+    return StatusCode::INTERNAL_ERROR;
```

Comment:
NOT_SUPPORTED? NOT_IMPLEMENTED would suggest that it is not implemented but it is just disabled.